code/modules/recycling/disposal/holder.dm

```diff
@@ -87,6 +87,13 @@
 /obj/structure/disposalholder/proc/try_expel(datum/move_loop/source, succeed, visual_delay)
 	SIGNAL_HANDLER
 	if(current_pipe || !active)
+		if(!(locate(/obj/structure/disposaloutlet) in get_turf(src)) && prob(15))
+			for(var/mob/living/H in(src))
+				H.apply_damage(1, BRUTE)
+				playsound(src.loc, 'sound/effects/wounds/crack2.ogg', 50, TRUE, FALSE)
+		if(!(locate(/obj/structure/disposaloutlet) in get_turf(src)) && prob(20))
+			for(var/mob/living/H in(src))
+				playsound(src.loc, 'sound/effects/clang.ogg', 30, TRUE, FALSE)
```

Comment:
Oh also you shouldn't use single letter variable names. Give it a name that describes what it is in the context of this proc.
What are we doing to H, who is H, etc. It's a chance to convey extra information about the code, don't let it drop.

Reply:
You are changing balance is the thing. Making disposal rides painful changes how people use them. Need to consider that and if/why you want it when you make a change is the thing. Intention doesn't affect intent.